ha/core/system_health/system_health.py

```diff
@@ -86,6 +90,145 @@ def get_status(self, component: str, component_id: str=None, **kwargs):
             Log.error(f"Failed reading status for component: {component} with Error: {e}")
             raise HaSystemHealthException("Failed reading status")
 
+    def get_status(self, component: CLUSTER_ELEMENTS = CLUSTER_ELEMENTS.CLUSTER.value, depth: int = 1, version: str = SYSTEM_HEALTH_OUTPUT_V1, **kwargs):
+        """
+        Return health status for the requested components.
+        Args:
+            component ([CLUSTER_ELEMENTS]): The component whose health status is to be returned.
+            depth ([int]): A depth of elements starting from the input "component" that the health status
+                is to be returned.
+            **kwargs([dict]): Variable number of arguments that are used as filters,
+                e.g. "id" of the input "component".
+        Returns:
+            ([dict]): Returns dictionary. {"status": "Succeeded"/"Failed"/"Partial", "output": "", "error": ""}
+                status: Succeeded, Failed, Partial
+                output: Dictionary with element health status
+                error: Error information if the request "Failed"
+        """
+
+        try:
+            component_id = None
+            if GET_SYS_HEALTH_ARGS.ID.value in kwargs and kwargs[GET_SYS_HEALTH_ARGS.ID.value] != "":
+                component_id = kwargs[GET_SYS_HEALTH_ARGS.ID.value]
+
+            # Get raw status starting from cluster
+            self._status_dict = self.get_status_raw(CLUSTER_ELEMENTS.CLUSTER.value)
+            # Remove any keys which are not for the health status.
+            ignore_keys = []
+            for key in self._status_dict:
```

Comment:
Should this code be part of get_status_raw? raw will filter out and will give only health related keys?

Reply:
Good suggestion! Done.